iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Разницу между авторизацией старого осм пользователя и регистрацией нового? Серьёзно не видишь разницы?
> On Mar 25, 2016, at 7:41 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm <https://github.com/mapsme/omim/pull/2511#discussion_r57461714>:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> При чем тут маркетологи? Ты сам эту разницу видишь? Я - не вижу.
> … <x-msg://41/#>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> 
>  <https://github.com/mapsme/omim/pull/2511#discussion_r57461616> <https://github.com/mapsme/omim/pull/2511#discussion_r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57457291> <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461616>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57461714>